genie-client/src/main/python/pygenie/jobs/presto.py

```diff
@@ -188,3 +188,25 @@ def session(self, name, value):
         self._set_command_option('--session', name, value)
 
         return self
+
+    @unicodify
+    @add_to_repr('append')
```

Comment:
Probably shouldn't call `@add_to_repr()` here because it'll also add to repr when calling `self.session()`.